scripts/sysman/device.yml

```diff
@@ -355,6 +355,46 @@ returns:
     - $X_RESULT_ERROR_INVALID_SIZE:
         - "The provided value of pCount is not big enough to store information about all the processes currently attached to the device."
 
+--- #--------------------------------------------------------------------------
+type: struct
+desc: "Sub Device Properties"
+version: "1.9"
+class: $sDevice
+name: $s_subdevice_properties_t
```

Comment:
can we start with experimental extensions for all interfaces changes in this PR? (all structs, APIs as experimental ) that gives us flexibility to evolve this going forward and fix any errors.

to covert this to extension, you need to add new `.rst` and `.yml` files. please check `scripts/sysman/rasState.yml` for example

Reply:
Moved new additions to experimental extensions. please check.